src/components/search-input/SearchInput.type.ts

```diff
@@ -0,0 +1,3 @@
+export interface FormData {
```

Comment:
인터페이스명 컨벤션 지켜주시면 좋을 것 같아요!

```suggestion
export interface IFormData {

Reply:
수정했습니다!